patches/minecraft/net/minecraft/util/EnumFacing.java.patch

```diff
@@ -1,46 +1,55 @@
 --- a/net/minecraft/util/EnumFacing.java
 +++ b/net/minecraft/util/EnumFacing.java
-@@ -109,7 +109,6 @@
+@@ -13,8 +13,6 @@
+ import net.minecraft.entity.Entity;
+ import net.minecraft.util.math.MathHelper;
+ import net.minecraft.util.math.Vec3i;
+-import net.minecraftforge.api.distmarker.Dist;
+-import net.minecraftforge.api.distmarker.OnlyIn;
```

Comment:
uh no

Reply:
It was in the original patch, probably to make sure no method goes OnlyIn without anyone noticing it, so I figured to keep it

Reply:
Actually sided imports are one.of the only ones i am fine with removing the imports. When i see that i know to just delete the imports and remove any erroring lines. Meaning remove ALL sidedness.